v21.2/restore.md

```diff
@@ -728,6 +747,13 @@ After the restore completes, add the `users` to the existing `system.users` tabl
 
 </section>
 
+## Known limitations
+
+* {% include {{ page.version.version }}/known-limitations/restore-aost.md %} [Tracking GitHub Issue](https://github.com/cockroachdb/cockroach/issues/53044)
+* To successfully [restore a table into a multi-region database](#restoring-to-multi-region-databases), it is necessary for the order and regions to match between the source and destination database. [Tracking GitHub Issue](https://github.com/cockroachdb/cockroach/issues/71071)
```

Comment:
I think this one could benefit from linking to your more full explanation on the Known Limitations page.  If I hadn't read that first I would not know what "the order ... to match" means